### ISSUE TITLE
pydantic.error_wrappers.ValidationError

### ISSUE THREAD

**Issue by corv89** [fief-dev/fief#258]
## Describe the bug

I am following the quick-start instructions at `https://docs.fief.dev/self-hosting/quickstart/`

## To Reproduce

`docker run -it --rm ghcr.io/fief-dev/fief:latest fief quickstart --docker`

## Expected behavior

The command will ask you to give an email address for the first admin user. It'll be automatically created when starting the server.

## Configuration

* Cloud or self-hosted: self-hosted
* If self-hosted, Fief version: v0.26.1 (latest)

## Additional context

Actual output:
```
% docker run -it --rm ghcr.io/fief-dev/fief:latest fief quickstart --docker
Unable to find image 'ghcr.io/fief-dev/fief:latest' locally
latest: Pulling from fief-dev/fief
4ee097f9a366: Pull complete
493e98a6d531: Pull complete
ed400aec434d: Pull complete
a18610e9d04b: Pull complete
195bd9e1cc4c: Pull complete
942bc5ab9ce1: Pull complete
c56efdd7dd86: Pull complete
daf9da8eda35: Pull complete
Digest: sha256:fe7fdcb9366ab2d5afcccc4569c4c117648bee68215baf031173f9cf1c301de0
Status: Downloaded newer image for ghcr.io/fief-dev/fief:latest
/usr/local/lib/python3.11/site-packages/pydantic/env_settings.py:289: UserWarning: directory "/run/secrets" does not exist
  warnings.warn(f'directory "{secrets_path}" does not exist')
Traceback (most recent call last):
  File "/usr/local/bin/fief", line 5, in <module>
    from fief.cli import app
  File "/usr/local/lib/python3.11/site-packages/fief/cli.py", line 20, in <module>
    from fief.services.user_manager import InvalidPasswordError, UserAlreadyExistsError
  File "/usr/local/lib/python3.11/site-packages/fief/services/user_manager.py", line 9, in <module>
    from fief import schemas
  File "/usr/local/lib/python3.11/site-packages/fief/schemas/__init__.py", line 1, in <module>
    from fief.schemas import (
  File "/usr/local/lib/python3.11/site-packages/fief/schemas/client.py", line 4, in <module>
    from fief.models.client import ClientType
  File "/usr/local/lib/python3.11/site-packages/fief/models/__init__.py", line 1, in <module>
    from fief.models.admin_api_key import AdminAPIKey
  File "/usr/local/lib/python3.11/site-packages/fief/models/admin_api_key.py", line 5, in <module>
    from fief.models.base import MainBase
  File "/usr/local/lib/python3.11/site-packages/fief/models/base.py", line 3, in <module>
    from fief.settings import settings
  File "/usr/local/lib/python3.11/site-packages/fief/settings.py", line 199, in <module>
    settings = Settings()
               ^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pydantic/env_settings.py", line 39, in __init__
    super().__init__(
  File "/usr/local/lib/python3.11/site-packages/pydantic/main.py", line 341, in __init__
    raise validation_error
pydantic.error_wrappers.ValidationError: 4 validation errors for Settings
secret
  field required (type=value_error.missing)
encryption_key
  field required (type=value_error.missing)
fief_client_id
  field required (type=value_error.missing)
fief_client_secret
  field required (type=value_error.missing)
```

**Comment by frankie567** [comment]
Indeed, thanks for the report, @corv89 🙏  I'll issue a release in a few minutes.

@all-contributors add @corv89 for bug

**Comment by frankie567** [comment]
Released in v0.26.2

**Comment by corv89** [comment]
Thanks for the quick turnaround, this release works
